lib/common/formats.js

```diff
@@ -110,6 +97,53 @@ module.exports = {
       .value().join('\n');
   },
 
+  /**
+   * Creates a LESS file with variable definitions based on the style dictionary
+   *
+   * @memberof Formats
+   * @kind member
+   * @example
+   * ```less
+   * @color-background-base: #f0f0f0;
+   * @color-background-alt: #eeeeee;
+   * ```
+   */
+  'less/variables': function(dictionary) {
+    var to_ret;
+    return fileHeader() + _.map(dictionary.allProperties, function (prop) {
+      to_ret = '@' + prop.name + ': ' + (prop.attributes.category==='asset' ? '"'+prop.value+'"' : prop.value) + ';';
+
+      if (prop.attributes.comment)
+        to_ret = to_ret.concat(' // ' + prop.attributes.comment);
+      return to_ret;
+    }).filter(function (strVal) { return !!strVal; }).join('\n');
+  },
+
+  /**
+   * Creates a SCSS file with variable definitions and helper classes for icons
+   *
+   * @memberof Formats
+   * @kind member
+   * @example
+   * ```less
+   * @content-icon-email: '\E001';
+   * .icon.email:before { content:@content-icon-email; }
+   * ```
+   */
+  'less/icons': function(dictionary, config) {
+    return fileHeader() + _.chain(dictionary.allProperties)
+      .filter(function(prop) {
+        return prop.attributes.category === 'content' && prop.attributes.type === 'icon';
+      })
+      .map(function(prop) {
+        var varName = '@' + prop.name + ': ' + prop.value + ';';
```

Comment:
we could probably do the same for this function since it's shared between SCSS & LESS